src/renderer/util/day.js

```diff
@@ -2,7 +2,7 @@ import dayjs from 'dayjs/esm'
 import relativeTime from 'dayjs/esm/plugin/relativeTime'
 import 'dayjs/esm/locale/en' // load on demand
```

Comment:
`en` is a built in locale.

There's no need to import it manually.

Reply:
@Jocs thanks, I'll modify it.